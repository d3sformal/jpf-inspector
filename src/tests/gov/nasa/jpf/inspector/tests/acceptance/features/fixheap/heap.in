create breakpoint it=any
sw bp
run
wait
# All of these commands should print an error message:
print #static[*]
print #static[*].member
print #heap[*].member
del bp 1
run
wait