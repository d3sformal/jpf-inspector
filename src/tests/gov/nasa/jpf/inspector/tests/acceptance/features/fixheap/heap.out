1 :  state=ENABLED hits=0 hitsTotal=0 instruction_type=any
INFO: JPF created and connected, SuT is started
INFO: Breakpoint hit: [1 :  state=ENABLED hits=1 hitsTotal=1 instruction_type=any]
	SuT (Thread=0) executes the gov/nasa/jpf/inspector/tests/acceptance/sut/Singlepath.java:0 - invokestatic gov.nasa.jpf.inspector.tests.acceptance.sut.Singlepath.main([Ljava/lang/String;)V
ERR: Static area entry discovery is not implemented.
ERR: Parse error: mismatched input '.' expecting <EOF>
#static[*].member
          ^
ERR: Parse error: mismatched input '.' expecting <EOF>
#heap[*].member
        ^
Breakpoint with ID 1 successfully deleted.
INFO: JPF Terminating